frontend/src/slices/auth/actions.ts

```diff
@@ -18,4 +18,14 @@ const signUp = createAsyncThunk<
   return authApi.signUp(registerPayload);
 });
 
-export { signUp };
+const getUser = createAsyncThunk<
```

Comment:
And also rename it everywhere (except store, there it will be just user)
```suggestion
const getAuthenticatedUser = createAsyncThunk<
```